File: problems/d/different/different.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    while True:
        try:
            a, b = [int(val) for val in input().split()]
            print(abs(a - b))
        except EOFError:
            break


if __name__ == '__main__':
    main()

File: problems/d/different/different.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <bits/stdc++.h>

using namespace std;

int main() {
    string line;
    while (getline(cin, line)) {
        stringstream ss(line);
        unsigned long long a, b;
        ss >> a >> b;

        long long difference = a - b;

        cout << abs(difference) << endl;
    }
}